corehq/apps/tour/templates/tour/config/new_blank_app.html

```diff
@@ -0,0 +1,63 @@
+{% load i18n %}
+<script type="text/javascript">
```

Comment:
This would be hard to simplify in general as each tour might want to do different things to the UI / highlight certain buttons, maybe `orphan: false` isn't necessary, or you want to utilize other callback functions. That part is already abstracted for us, and I see it as over-engineering to abstract it even further. This case is unique as the two tours are very similar, but in my opinion this is very little scaffolding for what is actually happening under the hood. Keeping it in javascript / django template is also nice because css ids stay where they're supposed to be (in the html and not in python), it's easy to implement callback functions (which can't be done with pure json), and you can utilize django translations. It's not like I didn't think about this.